README.md

```diff
@@ -0,0 +1,78 @@
+# RDS Scheduler
+
+[![Build_Status](https://circleci.com/gh/appvia/rds-scheduler.svg?style=svg)](https://circleci.com/gh/appvia/rds-scheduler) [![Docker Repository on Quay](https://quay.io/repository/appvia/rds-scheduler/status "Docker Repository on Quay")](https://quay.io/repository/appvia/rds-scheduler)
+
+Manage uptime schedules for RDS Instances and shutdown instances outside of working hours.
+
+All RDS instances are checked for a given tag, `appvia.io/rds-scheduler/uptime-schedule`, to determine whether they need to be managed according to a specified uptime schedule. If the tag is not found, no action is taken on that DB instance.
```

Comment:
AWS tag? (may be worth clarifying; probably obvious to most, but may not be to some).

Reply:
Yup sounds good, updated :)